src/ensembl/src/shared/components/radio/Radio.tsx

```diff
@@ -14,6 +16,9 @@ type Props = {
   classNames?: {
     radioInput?: string;
     radioLabel?: string;
+    radioWrapper?: string;
+    defaultRadio?: string;
+    checked?: any;
```

Comment:
Why `any`? Surely it's `string`

Reply:
i didnt quite understand this as well, when i put string, it throws a warning at the following line:
`{ [styles.checked]: option.value === props.selectedOption}`
And the error is 
`
(property) checked?: string | undefined
A computed property name must be of type 'string', 'number', 'symbol', or 'any'.ts(2464)
`
I didn't quite understand. Any idea why?

Reply:
Yes. With `checked` as an optional property, what it is saying is that checked can be `undefined`, and it doesn't want to set an object key as undefined. 

you can change this:
```
const radioClass = classNamesMeger(styles.defaultRadio, {
  [styles.checked]: option.value === props.selectedOption
});
```
to this:
```
const radioClass = classNamesMeger(styles.defaultRadio, 
  option.value === props.selectedOption ? styles.checked : undefined
);
```
then it won't complain.

Also, please change `any` in `props.options.map((option: any, index: number)` to `RadioOption`. Or, better yet, just remove the types from both option and index parameters — they are trivially inferred, and Typescript can figure them out by itself.

Reply:
@azangru thats all done now. Thanks for your help. can you please have a look and let me know if its all good (Dont forget to accept the PR if its all good :) ).
The check should also now be fixed, i have forgotten to push one file.